.travis.yml

```diff
@@ -5,6 +5,10 @@ php:
     - 7.2
     - nightly
 
+env:
+  - SYMFONY_VERSION="^3.3.0"
```

Comment:
In fact, this one will install the latest 3.x.y release, `3.3.*` is the way to go! :D